src/Uno.UI.RuntimeTests/Tests/Windows_UI_Composition/Given_CompositionNineGridBrush.cs

```diff
@@ -22,6 +22,8 @@ public class Given_CompositionNineGridBrush
 	[RunsOnUIThread]
 #if !__SKIA__
 	[Ignore("LoadedImageSurface is only implemented on skia")]
+#else
+	[Ignore("Test is incorrectly authored")]
 #endif
 	public async Task When_Source_Changes()
```

Comment:
The test was asserting a behavior that doesn't match WinUI and is now matched with this PR.

![image](https://github.com/unoplatform/uno/assets/31348972/1c62fbe0-9ab5-42f6-bbd3-418a9a6934c1)